From bd556157bacdd9741b920cf108d950114eb183b9 Mon Sep 17 00:00:00 2001
From: Tobias Rittig <tobias.rittig@t-online.de>
Date: Sun, 28 Jan 2018 12:45:31 +0100
Subject: [PATCH] #Fix building against tesseract4.0.0-alpha

---
 tesseract.pxd | 2 --
 tesserocr.pyx | 5 +++--
 2 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/tesseract.pxd b/tesseract.pxd
index 0582e9f..798d34e 100644
--- a/tesseract.pxd
+++ b/tesseract.pxd
@@ -219,8 +219,6 @@ cdef extern from "tesseract/baseapi.h" namespace "tesseract" nogil:
             OEM_LSTM_ONLY
             OEM_TESSERACT_LSTM_COMBINED
             OEM_DEFAULT
-            OEM_CUBE_ONLY
-            OEM_TESSERACT_CUBE_COMBINED
     ELSE:
         cdef enum OcrEngineMode:
             OEM_TESSERACT_ONLY
diff --git a/tesserocr.pyx b/tesserocr.pyx
index cce6681..e52e6a3 100644
--- a/tesserocr.pyx
+++ b/tesserocr.pyx
@@ -81,8 +81,9 @@ cdef class OEM(_Enum):
     IF TESSERACT_VERSION >= 0x040000:
         LSTM_ONLY = OEM_LSTM_ONLY
         TESSERACT_LSTM_COMBINED = OEM_TESSERACT_LSTM_COMBINED
-    CUBE_ONLY = OEM_CUBE_ONLY
-    TESSERACT_CUBE_COMBINED = OEM_TESSERACT_CUBE_COMBINED
+    ELSE:
+        CUBE_ONLY = OEM_CUBE_ONLY
+        TESSERACT_CUBE_COMBINED = OEM_TESSERACT_CUBE_COMBINED
     DEFAULT = OEM_DEFAULT
 
 
