diff --git a/CMakeLists.txt b/CMakeLists.txt
index 27e29e3..98b0bc7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -104,6 +104,10 @@ endif()
 #
 ###############################################################################
 
+find_file(STDINT_H stdint.h)
+get_filename_component(STDINT_INCLUDE_DIR "${STDINT_H}" DIRECTORY)
+include_directories(${STDINT_INCLUDE_DIR})
+
 set(AUTOCONFIG_SRC ${CMAKE_BINARY_DIR}/config_auto.h.in)
 set(AUTOCONFIG ${CMAKE_BINARY_DIR}/config_auto.h)
 
