diff --git a/vs2010/port/gettimeofday.cpp b/vs2010/port/gettimeofday.cpp
index b785245..cc9eafb 100644
--- a/vs2010/port/gettimeofday.cpp
+++ b/vs2010/port/gettimeofday.cpp
@@ -20,7 +20,7 @@
 #include <allheaders.h>
 #include "gettimeofday.h"
 
-int gettimeofday(struct timeval *tp, struct timezone *tzp) {
+int gettimeofday(struct timeval *tp, struct mytimezone *tzp) {
   l_int32 sec, usec;
   if (tp == NULL)
     return -1;
diff --git a/vs2010/port/gettimeofday.h b/vs2010/port/gettimeofday.h
index c34215b..65bdcf2 100644
--- a/vs2010/port/gettimeofday.h
+++ b/vs2010/port/gettimeofday.h
@@ -24,8 +24,8 @@
 #include          <winsock.h>    // timeval is defined in here.
 #endif
 
-typedef struct  timezone tz; 
+typedef struct  mytimezone tz; 
 
-int gettimeofday(struct timeval * tp, struct timezone * tzp);
+int gettimeofday(struct timeval * tp, struct mytimezone * tzp);
 
 #endif  // VS2008_PORT_GETTIMEOFDAY_H_
