diff --git a/cmake/templates/LeptonicaConfig.cmake.in b/cmake/templates/LeptonicaConfig.cmake.in
index d53904a..ad2b553 100644
--- a/cmake/templates/LeptonicaConfig.cmake.in
+++ b/cmake/templates/LeptonicaConfig.cmake.in
@@ -22,6 +22,8 @@
 
 include(${CMAKE_CURRENT_LIST_DIR}/LeptonicaTargets.cmake)
 
+get_filename_component(_leptonica_install_prefix "${CMAKE_CURRENT_LIST_DIR}/../../../" ABSOLUTE)
+
 # ======================================================
 #  Version variables:
 # ======================================================
@@ -36,7 +38,7 @@ SET(Leptonica_VERSION_PATCH     @VERSION_PATCH@)
 # ======================================================
 
 # Provide the include directories to the caller
-set(Leptonica_INCLUDE_DIRS @INCLUDE_DIR@)
+set(Leptonica_INCLUDE_DIRS "${_leptonica_install_prefix}/include/leptonica/")
 
 # ====================================================================
 # Link libraries:
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index ecc37c5..f6fa8d4 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -30,21 +30,25 @@ if (GIF_LIBRARY)
     target_link_libraries       (leptonica ${GIF_LIBRARY})
 endif()
 if (JPEG_LIBRARY)
+    include_directories         (${JPEG_INCLUDE_DIR})
     target_link_libraries       (leptonica ${JPEG_LIBRARY})
 endif()
 if (JP2K_FOUND)
     target_link_libraries       (leptonica ${JP2K_LIBRARIES})
 endif()
 if (PNG_LIBRARY)
+    include_directories         (${PNG_PNG_INCLUDE_DIR})
     target_link_libraries       (leptonica ${PNG_LIBRARY})
 endif()
 if (TIFF_LIBRARY)
+    include_directories         (${TIFF_INCLUDE_DIR})
     target_link_libraries       (leptonica ${TIFF_LIBRARY})
 endif()
 if (WEBP_FOUND)
     target_link_libraries       (leptonica ${WEBP_LIBRARIES})
 endif()
 if (ZLIB_LIBRARY)
+    include_directories         (${ZLIB_INCLUDE_DIR})
     target_link_libraries       (leptonica ${ZLIB_LIBRARY})
 endif()
 
