diff --git a/prog/dwalinearlow.3.c b/prog/dwalinearlow.3.c
index 486de22..44f28a4 100644
--- a/prog/dwalinearlow.3.c
+++ b/prog/dwalinearlow.3.c
@@ -37,6 +37,8 @@
 
 #include "allheaders.h"
 
+#pragma optimize("", off)
+
 static void  fdilate_3_0(l_uint32 *, l_int32, l_int32, l_int32, l_uint32 *, l_int32);
 static void  ferode_3_0(l_uint32 *, l_int32, l_int32, l_int32, l_uint32 *, l_int32);
 static void  fdilate_3_1(l_uint32 *, l_int32, l_int32, l_int32, l_uint32 *, l_int32);
@@ -16984,3 +16986,4 @@ l_int32             wpls29, wpls30, wpls31;
     }
 }
 
+#pragma optimize("", on)
